sdk/core/Azure.Core/tests/TestFramework/RecordMatcher.cs

```diff
@@ -83,11 +88,28 @@ public virtual RecordEntry FindMatch(Request request, IList<RecordEntry> entries
                 }
             }
 
-
             throw new InvalidOperationException(GenerateException(request.Method, uri, headers, bestScoreEntry));
         }
 
-        public virtual bool IsEquivalentResponse(RecordEntry entry, RecordEntry otherEntry)
+        public virtual bool IsEquivalentRecord(RecordEntry entry, RecordEntry otherEntry) =>
+            IsEquivalentRequest(entry, otherEntry) &&
```

Comment:
Does it matter that request it the same too?

Reply:
Yes - if I changed the URI but it generated an identical response, I want the recording to be serialized.  (I hit this specifically because I tried to change the URI to force a recording to serialize and was stumped when nothing happened.)